### DISCUSSION TITLE
Input masking doesn't work in some case

### DISCUSSION THREAD

**Discussion by facido** [alpinejs/alpine#3291] [7. Bugs (archived)]
The following line fails:
`<input x-data x-mask="(+82)999-9999-9999" value="01642321982">`

It should show as follows:
(+82)016-4232-1982

However, the result is as follows:
(+82)016-4321-92

**Comment by joshhanley** [comment]
This issue has been converted to a discussion as we are now using discussions to track bugs and ideas.
See this announcement for more details https://github.com/alpinejs/alpine/discussions/1423
